src/utils/translation.ts

```diff
@@ -14,6 +15,7 @@ export const supportedLanguages = new Map([
   ["ru", "русский"],
   ["es", "español"],
   ["fr", "français"],
+  ["pt", "português"],
```

Comment:
Based on my research it's Portuguese (Brazil), also you set `ptBR` as `data-fns/locale`